### ISSUE TITLE
Understanding 1.4.13 Content on hover or focus (Level AA)

### ISSUE THREAD

**Issue by nrhsinclair** [w3c/wcag-eo#4]
Post suggested revisions here.

**Comment by cochfarf** [comment]
Under the Dismissable section, second paragraph is the word "Alternatively" necessary?

**Comment by Amace41** [comment]
In the Intent heading, first paragraph:
Suggest changing the "in coordination with" to "automatically on keyboard focus or pointer hover...."
 Same heading, list item 3:
Spelling error "intefere" should be "interfere"

**Comment by RachelComerford** [comment]
**Under Intent heading, Item 3, change "do" to "perform"?**
Current: the new content may intefere with a user's ability to do a task
Proposed: the new content may intefere with a user's ability to perform a task

**Under Intent heading:**
"There are usually more predictable and accessible means of adding content to the page, which authors are recommended to employ."
- This is good information but more helpful with links to recommended solutions/approaches that are alternatives to this approach.

**Under Note**
I had trouble parsing the intention of the note - it seemed to say that the SC covered more than just the component but also not? I wanted to suggest edits but I wasn't sure what the intended message was.

**Hoverable**
Current: The intent of this condition is to ensure that additional content which may appear on hover of a target may also be hovered itself. 
Proposed: The intent of this condition is to ensure that when additional content appears as a hover because a pointer has come into contact with a trigger on the page, keeping the pointer within the content within the hover is sufficient to keep the hover open.

**Persistent**
Current: The user removes hover or focus from the trigger and the additional content, consistent with the typical user experience;
- Do we want to add 'to satisfy the Hoverable condition' ?

**Benefits**
Current: users with low pointer accuracy will be able to more easily dismiss unintentionally-triggered additional content
Grammatical Correction: Users with low pointer accuracy will be able to more easily dismiss unintentionally-triggered additional content.

**Comment by bakkenb** [comment]
**Under "Intent" heading.**
Drop the word "must" form this sentence (not needed)...
> The intent of this success criterion is to ensure that authors who cause additional content to appear and disappear in this manner must design the interaction in such a way that users can:

To me, the NOTE was clear. Basically saying that something that is not visible becomes visible AND is the trigger is not what this SC is about. Like if you tab to a "Skip" link and it shows up, but does not cover anything or go away until you tab away. IMO it's complicated, but doesn't need rewording.

**Under "Hoverable" heading**
Current: The intent of this condition is to ensure that additional content which may appear on hover of a target may also be hovered itself.
Proposed: The intent of this condition is to ensure that the additional content that appears on hover remains when continuing to hover over the trigger OR over the additional content.

**Comment by a11ycob** [comment]
> 
> 
> In the Intent heading, first paragraph:
> Suggest changing the "in coordination with" to "automatically on keyboard focus or pointer hover...."
> Same heading, list item 3:
> Spelling error "intefere" should be "interfere"

Amanda, if we remove "automatically" as well so that it reads "Additional content that appears and disappears _on_ keyboard focus or pointer hover..." do you think it still reads well?

**Comment by a11ycob** [comment]
> 
> 
> **Under "Intent" heading.**
> Drop the word "must" form this sentence (not needed)...
> 
> > The intent of this success criterion is to ensure that authors who cause additional content to appear and disappear in this manner must design the interaction in such a way that users can:
> 
> To me, the NOTE was clear. Basically saying that something that is not visible becomes visible AND is the trigger is not what this SC is about. Like if you tab to a "Skip" link and it shows up, but does not cover anything or go away until you tab away. IMO it's complicated, but doesn't need rewording.

I agree with Rachel that the note section could use a bit of clarity. I took a stab at tightening up the language. We can always revert back to the original if the group feels the change is not helpful.

> **Under "Hoverable" heading**
> Current: The intent of this condition is to ensure that additional content which may appear on hover of a target may also be hovered itself.
> Proposed: The intent of this condition is to ensure that the additional content that appears on hover remains when continuing to hover over the trigger OR over the additional content.

Added this to the doc.

**Comment by Amace41** [comment]
Thanks Chris. Happy to drop the 'automatically', the message is still clear so it's probably a little redundant.
